www/src/components/showcase-details.js

```diff
@@ -255,15 +256,13 @@ const ShowcaseDetails = ({ parent, data, isModal, categories }) => (
                 <meta
                   property="og:image"
                   content={`https://www.gatsbyjs.org${
-                    data.sitesYaml.childScreenshot.screenshotFile
-                      .childImageSharp.resize.src
+                    data.sitesYaml.childScreenshot.screenshotFile.publicURL
```

Comment:
Hey @abhijithvijayan! Thanks for the PR! 🤗

I just gave this a spin, and `publicURL` here will always be `undefined`, unfortunately. :(

This is because `data.sitesYaml` is coming in from https://github.com/gatsbyjs/gatsby/blob/master/www/src/templates/template-showcase-details.js#L96-L130. You added `publicURL` to the `StaticQuery` that takes care of pulling in the screenshot thumbnails for the previous and next navigation shown (only) in the "showcase site" modal.

We were misleading you in https://github.com/gatsbyjs/gatsby/issues/14469#issuecomment-499029287—apologies for that!

Since I'm at it, I'll take the liberty and push a fix so we can get this merged!

If I was unclear in explaining why this doesn't work as you expected it, maybe my follow-up commits will help; if you have any questions, don't hesitate to ask! And please 🙏 don't let this discourage you from contributing! ❤️

Reply:
Thank You